docs/en/guides/Plugin-test.md

```diff
@@ -612,12 +612,13 @@ stage('Run Agent Plugin Tests') {
 Find the button 'detail' of your Workload in the PR page. Enter to the page and get the elapsed time of your task.
 
 ### Workload 1
-#### Group 1 (2164.287s)
+#### Group 1 (2635.00s)
 scenario name | versions | elapsed time (sec)
 ---|---|---
-apm-toolkit-trace | 1 | 84.69
-jetty 9.x | 63 | 1970.88
-netty-socketio 1.x | 4 | 108.70
+elasticsearch-6.x-scenario | 7 | 326.00
```

Comment:
It seems to be inconsistent with jenkinsfile-agent-test-4

Reply:
Yes. @aderm Your test has been added to workload 4, rather than workload 1. Please update the document by following your recent test result.

> elasticsearch-6.x-scenario 6.7.1-6.8.4 (7) . - 4m 33s

Reply:
Ok, this part is to be remembered, just update it after the test is finished, just right.